src/mpl/configure.ac

```diff
@@ -855,11 +855,42 @@ if test "$ac_cv_func_usleep" = "yes" ; then
     PAC_FUNC_NEEDS_DECL([#include <unistd.h>],usleep)
 fi
 
-AC_CHECK_FUNCS(backtrace_symbols)
-AC_CHECK_LIB(backtrace, backtrace_create_state)
-AC_CHECK_LIB(unwind, unw_backtrace)
+# Check for libbacktrace support
+AC_LINK_IFELSE([
```

Comment:
I don't think this test will ever succeed in its current form, since we don't tell the linker anywhere that we need `-lbacktrace`.

Reply:
Right.  This needs a PUSH/POP for LIBS.  Something like this:

```
PAC_PUSH_FLAG([LIBS])
AC_CHECK_LIB(backtrace, backtrace_create_state)
AC_CHECK_LIB(unwind, unw_backtrace)
AC_LINK_IFELSE([
...
```

Reply:
Thanks! Should be fixed now